.github/workflows/WE2E_tests.yaml

```diff
@@ -0,0 +1,123 @@
+# This workflow uses runs the CI for the Regional Workflow
+name: Run WE2E Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - rrfs_ci
+    types: 
+      - labeled
+
+# Use custom shell with -l so .bash_profile is sourced which loads intel/oneapi/setvars.sh, and lmod path 
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
```

Comment:
I don't understand why you used a long series of jobs here instead of just simply a list of steps.  it seems far more complex than needed, so I'm curious what your thinking there was.  If you simply use steps, then any step that fails kills the job.  I don't see what the advantage of having each of these as separate jobs is.

Reply:
One of the reasons I chose to use jobs instead of steps is for Github Actions web interface when viewing the workflow. If there is only one job, you don't get any detail as to what failed on first glance, or what the workflow entails.  The more important reason is when submitting the matrix of WE2E tests as a job, obtaining updates via `./get_single_expt_status.sh` on each experiment will run in parallel. 

I can certainly combine the first couple of jobs into one job with multiple steps if you think that would be the best practice.

Reply:
It just seemed a lot more complicated than necessary with the capturing of output and using it for inter-job dependencies when steps would accomplish that without any of that extra stuff to glue things together.

Reply:
I think your last update has made these workflows a lot cleaner and simpler to understand.  Thank you.